packages/client/src/client.js

```diff
@@ -516,6 +516,21 @@ export class PercyClient {
     return comparison;
   }
 
+  async sendFailedEvents(buildId, { errorKind = 'sdk', client = null, clientVersion = null, cliVersion = null, errorMessage = null } = {}) {
+    validateId('build', buildId);
+    this.log.debug('Sending FailedEvents');
+    return this.post(`builds/${buildId}/failed-events`, {
```

Comment:
does this only send failed events ? so we can't calculate percentage ? we were going to make a reusable instrumentation endpoint right ?